Bug: Fix lint warning on ConfirmEmail.js react component

### Overview
The following linter warning is reported when the linter is run on the client module:
```
C:\git\hackforla\tdm-calculator\client\src\components\Authorization\ConfirmEmail.js
  42:6  warning  React Hook useEffect has missing dependencies: 'history' and 'toast'. Either include them or remove the dependency array 
 react-hooks/exhaustive-deps
```

### Action Items
- Fix it.

### Resources/Instructions

--- a/client/src/components/Authorization/ConfirmEmail.js
+++ b/client/src/components/Authorization/ConfirmEmail.js
@@ -39,7 +39,7 @@ const ConfirmEmail = props => {
     if (token) {
       confirmEmail(token);
     }
-  }, [token]);
+  }, [token, toast, history]);
 
   return confirmResult.success ? (
     <Redirect to={`/login/${confirmResult.email}`} />

